test/mapmatch.cc

```diff
@@ -1131,8 +1131,8 @@ TEST(Mapmatch, test_discontinuity_duration_trimming) {
   std::vector<int> test_ans_num_routes{2, 2, 2};
   std::vector<std::vector<int>> test_ans_num_legs{{1, 2}, {1, 2}, {1, 2}};
   std::vector<std::vector<float>> test_ans_leg_duration{{26.459, 0.97, 0.454},
-                                                        {48.6, 0.64, 0.961},
-                                                        {220.778, 2.431, 1.899}};
+                                                        {48.159, 0.64, 0.961},
+                                                        {93.388, 2.431, 1.899}};
```

Comment:
By the way, I found out why the difference is so large here. The route has a number of turns. "Utrecht tiles" data does not have traffic information so duration was multiplied by density factor, as a result there were durations like 15, 20 seconds, sometimes even bigger. With several turns it summed up to >100 seconds difference